docs/content/v2.12/admin/yb-admin.md

```diff
@@ -1618,6 +1469,159 @@ The CDC bootstrap ids are the ones that should be used with [`setup_universe_rep
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
+{{< /warning >}}
+
+{{< note title="Tip" >}}
+
+To display a list of tables and their UUID (`table_id`) values, open the **YB-Master UI** (`<master_host>:7000/`) and click **Tables** in the navigation bar.
+
+{{< /note >}}
+
+**Example**
+
+```sh
+./bin/yb-admin \
+    -master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 \
+    setup_universe_replication e260b8b6-e89f-4505-bb8e-b31f74aa29f3 \
+    127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+    000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
+```
+
+#### alter_universe_replication
+
+Changes the universe replication for the specified source universe. Use this command to:
+
+* Add or remove tables in an existing replication UUID
+* Modify the source master addresses
+
+If no tables have been configured for replication, use [setup_universe_replication](#setup-universe-replication).
+
+To check if any tables are configured for replication, use [list_cdc_streams](#list-cdc-streams).
+
+**Syntax**
+
+Use the `set_master_addresses` subcommand to replace the source master address list. Use this if the set of masters on the source changes:
+
+```sh
+yb-admin -master_addresses <target_master_addresses> \
+    alter_universe_replication <source_universe_uuid>_<replication_name> \
+    set_master_addresses <source_master_addresses>
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name of the replication to be altered.
+* *source_master_addresses*: Comma-separated list of the source master addresses.
+
+Use the `add_table` subcommand to add one or more tables to the existing list:
+
+```sh
+yb-admin -master_addresses <target_master_addresses> \
+    alter_universe_replication <source_universe_uuid>_<replication_name> \
+    add_table [ <comma_separated_list_of_table_ids> ] \
+    [ <comma_separated_list_of_producer_bootstrap_ids> ]
+```
+
+* *target_master_addresses*: Comma-separated list of target YB-Master hosts and ports. Default value is `localhost:7100`.
+* *source_universe_uuid*: The UUID of the source universe.
+* *replication_name*: The name of the replication to be altered.
+* *comma_separated_list_of_table_ids*: Comma-separated list of source universe table identifiers (`table_id`).
+* *comma_separated_list_of_producer_bootstrap_ids*: Comma-separated list of source universe bootstrap identifiers (`bootstrap_id`). Obtain these with [bootstrap_cdc_producer](#bootstrap-cdc-producer-comma-separated-list-of-table-ids), using a comma-separated list of source universe table IDs.
+
+{{< warning title="Important" >}}
+Enter the source universe bootstrap_ids in the same order as their corresponding table_ids.
```

Comment:
Enter the source universe bootstrap IDs in the same order as their corresponding table IDs.